stock_putaway_product_form/views/product.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3). -->
+<odoo>
+    <record id="view_template_form_putaway" model="ir.ui.view">
+        <field name="name">product.template.product.form</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view"/>
+        <field name="arch" type="xml">
```

Comment:
Please add field to view
`
<field name="groups_id" eval="[(4,ref('stock.group_stock_manager'))]"/>
`
The users could be in stock.group_adv_location but no in stock security groups and crahs when open a product form